app/controllers/programmes_controller.rb

```diff
@@ -27,8 +34,12 @@ def sessions
           .map do |location|
             Session.new(team: current_user.team, location:, academic_year:)
           end
+    @unscheduled_sessions.sort_by! { |session| session.location.name }
 
-    @completed_sessions = sessions_for_programme.completed
+    @completed_sessions =
+      sessions_for_programme.completed.sort_by do |session|
+        [session.dates.first.value, session.location.name]
```

Comment:
```suggestion
        [session.dates.first.value, session.location&.name]
```

Reply:
Fixed now. I debated whether to also push this sort into the database but held off for now for expediency. We can probably sweet it up when we revisit this code or if we discover it's not performant.

Reply:
Yeah, that makes sense!